jobs/stress-ng.yaml

```diff
@@ -1,7 +1,24 @@
 suite: stress-ng
 testcase: stress-ng
+category: benchmark
 
-kmsg:
```

Comment:
The kmsg monition has been included in default monitor set with a benchmark category.

#cat program_list 
kmsg
uptime
iostat
heartbeat
vmstat
numa-numastat
numa-vmstat
numa-meminfo
proc-vmstat
proc-stat
meminfo
slabinfo
interrupts
lock_stat
latency_stats
softirqs
bdi_dev_mapping
diskstats
nfsstat
cpuidle
cpufreq-stats
turbostat
sched_debug
perf-stat
mpstat
perf-profile
stress-ng